ASF-slaves.txt

```diff
@@ -7,7 +7,7 @@ Additional compute resource donations for the Apache Cassandra project are
 appreciated. If you have questions about compute resource donations, ask
 on the dev mailing list <dev@cassandra.apache.org>.
 
-The installed OS software is the stock online.net Ubuntu 16.04 LTS amd64 image
+The installed OS software is the stock online.net Ubuntu 18.04 LTS amd64 image
```

Comment:
1) After launching Ubuntu 18.04 LTS the next step was to create a Jenkins user and group using /home/jenkins

2) At this point I mounted the 500 GB data volume to /home/jenkins and added it to fstab so it will be there on startup.  Best to just follow these directions: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-using-volumes.html.

3)Add the Jenkins ssh public key into the users ~/.ssh/authorized keys. (key is here: https://github.com/apache/infrastructure-puppet/blob/deployment/modules/build_slaves/files/jenkins.pub)

Reply:
I'm confused about what you want me to correct here.

The instructions to create the jenkins user, and to add the key to authorized keys, is there already.

Check out my update and let me know if anything needs to be updated…

And… did you still need to uninstall `bind9` ?